test/Orleans.Providers.Legacy.Test/Bootstrap/BootstrapProviderTests.cs

```diff
@@ -154,5 +168,30 @@ public BootstrapProvidersTests(ITestOutputHelper output, Fixture fixture)
             }
             return -1;
         }
+
+        private async Task<int> GetAllSiloProviderNames(string providerName)
+        {
+            var mgmt = this.fixture.GrainFactory.GetGrain<IManagementGrain>(0);
+            // request provider InitCount on all silos in this cluster
+            object[] results = await mgmt.SendControlCommandToProvider(typeof(GrainCallBootstrapper).FullName, GrainCallBootstrapperName, (int)MockBootstrapProvider.Commands.InitCount, null);
```

Comment:
Function name isn't expressing the actual behavior. This is more like `GetGrainCallBootstrapperProviderCountOnAllSilos`.

Reply:
And is this method still in use? It seems being replaced by the method below